nestkernel/nestmodule.cpp

```diff
@@ -905,95 +905,97 @@ NestModule::MemoryInfoFunction::execute( SLIInterpreter* i ) const
    subnet is continued.
 
    Example:
-   SLI ] /iaf_neuron Create
+   SLI ] /iaf_psc_alpha Create
    SLI [1] /iaf_cond_alpha 10 Create
    SLI [2] /dc_generator [2 5 6] LayoutNetwork
-   SLI [3] [0] 1 PrintNetwork
-   +-[0] Subnet Dim=[12]
+   SLI [3] 0 1 PrintNetwork
+   +-[0] root dim=[12]
+   SLI [3] 0 2 PrintNetwork
+   +-[0] root dim=[12]
       |
-      +- iaf_neuron [1]
-      +- lifb_cond_neuron [2]..(10)..[11]
-      +-[12] Subnet Dim=[2 5 6]
-   SLI [3] [0] 2 PrintNetwork
-   +-[0] Subnet Dim=[12]
+      +- [1] iaf_psc_alpha
+      +- [2]...[11] iaf_cond_neuron
```

Comment:
I must have been very focused on `iaf_psc_alpha`! Thanks for the review, I will look at it now.